frontend/src/components/collab/ProblemDescription.tsx

```diff
@@ -0,0 +1,84 @@
+import Question from "@/types/question";
+import { FC } from "react";
+import ComplexityChip from "../question/ComplexityChip";
+import { Divider } from "@nextui-org/react";
+import parse from "html-react-parser";
+
+interface ProblemDescriptionProps {
+  question: Question;
+}
+
+const ProblemDescription: FC<ProblemDescriptionProps> = ({ question }) => {
+  return (
+    <div className="flex px-0 py-4 h-[calc(100vh-94px)] overflow-y-auto">
+      <div className="w-full px-5">
+        {/* Question title */}
+        <div className="flex space-x-4">
+          <div className="flex-1 mr-2 text-xl text-white font-semibold">
+            {question.title}
+          </div>
+        </div>
+        {/* Question complexity */}
+        <div className="flex items-center mt-3">
+          <div className="px-2.5 py-1 text-sm font-medium capitalize">
+            <ComplexityChip complexity={question.complexity} size="sm" />
+          </div>
+          {question.author && (
+            <div className="px-2.5 py-1 text-sm font-medium text-gray-300">
+              From <span className="font-semibold">{question.author}</span>
+              <sup>&copy;</sup>
+            </div>
+          )}
+        </div>
+
+        <Divider className="mt-4 mb-2" />
+
+        {/* Question description */}
+        <div className="flex mt-3">
+          <div className="text-md text-white text-justify">
+            {parse(question.description!)}
+          </div>
+        </div>
+
+        {/* Question examples */}
+        <div className="mt-4">
+          {question.examples?.map((example, index) => (
+            <div key={index}>
+              <p className="font-medium text-white ">Example {index + 1}: </p>
+              <div className="">
+                <pre className="bg-gray-600 bg-opacity-50 my-4 p-4 rounded-lg text-white text-sm whitespace-pre-wrap">
+                  <strong className="text-white text-base">Input: </strong>{" "}
+                  {example.input}
+                  <br />
+                  <strong className="text-white text-base">Output: </strong>
+                  {example.output}
+                  <br />
+                  {example.explanation && (
+                    <>
+                      <strong className="text-white text-base">
+                        Explanation:{" "}
+                      </strong>
+                      {example.explanation}
+                    </>
+                  )}
+                </pre>
+              </div>
+            </div>
+          ))}
+        </div>
```

Comment:
hmm... as there might be images inserted for the examples, so my idea here is that the example detail, with the images, in/out, and explanation will all be in the description field, the values can be used for test cases later on instead. so not sure if you want to render it separetly here.

Reply:
Hmm, I understand your concern, but we don't know what is available from Leetcode/whatever platform. So I think we just stick to our design for now, and leave the change in the future ba. For this, I will add `parse()` for each input and output in examples.